packages/client/components/PokerVotingAvatarGroup.tsx

```diff
@@ -14,14 +14,14 @@ const NoVotesHeaderLabel = styled('div')({
   fontSize: 14,
   fontWeight: 600,
   lineHeight: '24px',
-  paddingLeft: 2 // same as avatar border
+  paddingLeft: 3 // same as avatar border
```

Comment:
could you make a constEnum like `PokerCards.PokerAvatarBorderWidth`?
then, we can use that instead of comments